<launch>
  <param name="Control_Update_dt" type="double" value="0.01"/>
  <param name="gain_radius" type="double" value="2"/>
  <param name="gain_theta" type="double" value="60"/>

  <include file="$(find image_gui)/launch/image_gui.launch" />
  <include file="$(find joystick_commands)/launch/joystick_commands.launch" />
  <include file="$(find plate_tf)/launch/plate_tf.launch" />
  <include file="$(find stage_tf)/launch/stage_tf.launch" />

  <node pkg="save_data" type="SaveBags.py" name="SaveBags" />

  <param name="robot_velocity_max" type="double" value="60"/>
  <node pkg="flyatar_avatar" type="SetpointControl.py" name="SetpointControl" />

  <!-- <param name="contour_count_max" type="double" value="2"/> -->
  <!-- <param name="image_processor_display_images" type="boolean" value="false"/> -->
</launch>
